provider/test-programs/network/Pulumi.yaml

```diff
@@ -5,22 +5,30 @@ resources:
   exampleNetwork:
     type: gcp:compute/network:Network
     properties:
-      name: pulumi-network
+      name: mynetwork2
+        # Randomizing the name fails with "  warning: Error downloading plugin: unknown version for plugin str"
+        # fn::invoke:
+        #   function: str:replace
+        #   arguments:
+        #     old: "-"
+        #     new: ""
+        #     string: network-${pulumi.stack}
+        #   return: result
       autoCreateSubnetworks: false  # We'll create our own subnetwork
 
   exampleSubnetwork:
     type: gcp:compute/subnetwork:Subnetwork
     properties:
-      name: pulumi-subnetwork
+      name: pulumi-subnetwork2
```

Comment:
The names are not randomized means that if the previous tests' run did not clean up we get name collisions.